src/code/InstallPkgParams.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.PowerShell.PowerShellGet.UtilClasses;
+using NuGet.Versioning;
+using System;
+using System.Collections;
+using System.Management.Automation;
+
+public class InstallPkgParams
+{
+    public string Name { get; set; }
+    public VersionRange Version { get; set; }
+    public string Repository { get; set; }
+    public bool AcceptLicense { get; set; } 
+    public bool Prerelease { get; set; }
+    public ScopeType Scope { get; set; }
+    public bool Quiet { get; set; }
+    public bool Reinstall { get; set; }
+    public bool Force { get; set; }
+    public bool TrustRepository { get; set; }
+    public bool NoClobber { get; set; }
+	public bool SkipDependencyCheck { get; set; }

```

Comment:
```suggestion
public bool SkipDependencyCheck { get; set; }

```
This is using tab character for indent.  Please be sure to conform to tab == whitespace convention so our source file formats are consistent.